### ISSUE TITLE
BS4: Clean up Practical Support Tab Layout

### ISSUE THREAD

**Issue by lomky** [DCAFEngineering/dcaf_case_management#1821]
* **What are we trying to do?**
Make the Practical Support page look pretty on BS4.

* **What feature or behavior is this required for?**
\~\*\~Design~*~

#### A bit cluttered
![Screenshot from 2019-10-18 19-25-08](https://user-images.githubusercontent.com/6129479/67134439-278c8680-f1e0-11e9-98bd-c6817a62d376.png)

#### Oh no my columns
![Screenshot from 2019-10-18 19-25-30](https://user-images.githubusercontent.com/6129479/67134438-278c8680-f1e0-11e9-9aa1-d579c306035e.png)

* **How to fix it!**

Checkout the `bs4-2` branch and make a new branch off it. Fix from there, then PR back to `bs4-2`.

```
git pull
git checkout bs4-2
git checkout -b bs4-2_pretty_up_practical_support
# Do the work!
git push
# Submit a PR!
```

**Comment by colinxfleming** [comment]
To see these, take the following steps:

* Log in to the app and select the DC line to go to the case manager dashboard
* Click on Patient 0's name to go to that patient's edit view
* Click the practical support link to see the practical support forms
* Resize them. They look really cramped!

Don't worry too much about cell-phone sized stuff, but do use your discretion for making this look nice!

These files are in: app/views/patients/_practical_support.html.erb, and the contents are in app/views/practical_supports/entries.html.erb and app/views/practical_supports/new.html.erb

**Comment by colinxfleming** [comment]
(Please leave for tech rebalanced volunteers!)

**Comment by Elexc** [comment]
Working on this!

**Comment by colinxfleming** [comment]
resolved by #1838
